roles/advanced-core/advanced_dhcp_server/tasks/main.yml

```diff
@@ -44,7 +44,7 @@
 - name: Template >> /etc/dhcp/dhcpd.conf
   template:
     src: dhcpd.conf.j2
-    dest: /etc/dhcp/dhcpd.conf
+    dest: /etc/dhcp/dhcpd.conf.new
```

Comment:
The unitary tests are failing due to ` .new` suffix added in all configuration files ?
Why renaming all configuration files ?

Also the following configuration will fail:

```
# cat /etc/dhcp/dhcpd.conf.new
[...]

## NETWORKS AND NODES

include "/etc/dhcp/dhcpd.networks.conf";
```

Reply:
Oups, thanks a lot !
This is a bug. Correcting.